src/index.html

```diff
@@ -11,7 +11,191 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend>
+          Personal information
+        </legend>
+        <div class="form-field">
+          <label for="surname">
+            Surname:
+          </label>
+          <input
+            id="surname"
+            type="text"
+            name="Surname"
+            autocomplete="off"
+          >
+        </div>
+        <div class="form-field">
+          <label for="name">
+            Name:
+          </label>
+          <input
+            id="name"
+            type="text"
+            name="Name"
+            autocomplete="off"
+          >
+        </div>
+        <div class="form-field">
+          <label for="age">
+            How old are you?
+          </label>
+          <input
+            type="number"
```

Comment:
let's make some spacing between form-fields in order it was easier to read your code
```suggestion
        <div class="form-field">
          <label for="name">
            Name:
          </label>
          <input
            id="name"
            type="text"
            name="Name"
            autocomplete="off"
          >
        </div>
        
        <div class="form-field">
          <label for="age">
            How old are you?
          </label>
          <input
            type="number"
```